job_templates/ansible_roles_-_install_from_git.erb

```diff
@@ -0,0 +1,29 @@
+<%#
```

Comment:
Should be fixed now

Reply:
I see this has been now changed to single resource (name and example in description) but the template still accepts multiple items, what is the desired usage? If it allows multiple repos, should the input name be plural?

Reply:
It only works with a single repo. The reason is that you can't simply `git clone myrepo /etc/ansible/roles`. That returns an error `fatal: destination path '/etc/ansible/roles' already exists and is not an empty directory`. The location must be something like `/etc/ansible/roles/something` so that we're able to clone.

I wondered how to do this parsing the input, but as an example:

1. `git_repositories = puppet-foreman
openshift-ansible
kubernetes-ansible`
2. There's only one location that can be specified so far, what should it be? `location = /etc/ansible/roles/openshift`
3. Since there's only one location, it doesn't make sense for all repositories to use the same one.

I'm just realizing we could use:

```yaml
  with_items:
    - { git_repository: 'somerepo', location: 'somelocation' }
    - { git_repository: 'somerepo', location: 'somelocation' }
```

and build this structure with 2 inputs that accept multiple values.